financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerPreviewParameterProvider.kt

```diff
@@ -17,7 +18,8 @@ internal class LinkAccountPickerPreviewParameterProvider :
     PreviewParameterProvider<LinkAccountPickerState> {
     override val values = sequenceOf(
         canonical(),
-        accountSelected()
+        accountSelected(),
+//        repairableAccountSelected(),
```

Comment:
is this intended?

Reply:
reverted (that test is coming in a future PR)